lang/erlang/Makefile

```diff
@@ -35,7 +35,7 @@ define Package/erlang/Default
   SECTION:=lang
   CATEGORY:=Languages
   TITLE:=Erlang/OTP programming language
-  URL:=http://www.erlang.org/
+  URL:=https://www.erlang.org/
```

Comment:
This change is described where?

Reply:
You're right, I've PR description has been updated